### PR TITLE
[Publisher][Design Update][Metric Settings] Update layout of Metric Availability & Breakdowns page (2/3)

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Updates the layout of Metric Availability & Breakdowns page to match the [Figma designs](https://www.figma.com/file/dvX3yg7FMGJSWFaqHWrWgQ/Justice-Counts-Publisher?type=design&node-id=9418%3A3634&mode=design&t=RMICoZJlXhkvO6rU-1).

The changes are mostly styling changes - no-op with regards to functionality.

Note: there is a treatment for the annual frequency custom month dropdown incoming, will update this PR once that design is ready.

Demo: https://mahmoudtest---publisher-web-b47yvyxs3q-uc.a.run.app/

## Related issues

Contributes to #1167 

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: common/components/CheckboxOptions/CheckboxOptions.tsx [hunk 1]
```diff
@@ -32,15 +39,22 @@ export const CheckboxOptions: React.FC<CheckboxOptionsProps> = ({
...
-        id={key}
-        type="checkbox"
-        checked={checked}
-        onChange={() => onChange({ key, checked })}
-      />
-      <Styled.CheckboxLabel>{label}</Styled.CheckboxLabel>
-    </Styled.CheckboxOptionsWrapper>
-  ));
+  return options.map(
+    ({ key, label, checked, disabled, icon, onChangeOverride }) => (
```
Comment (mxosman) [inline common/components/CheckboxOptions/CheckboxOptions.tsx:43]: Added some more parameters I needed and am using them below.

File: publisher/src/components/MetricsConfiguration/Configuration.tsx [hunk 2]
```diff
@@ -51,76 +54,79 @@ function Configuration() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [metricConfigPage]);
 
+  useEffect(() => {
+    document.body.style.overflow = isRaceEthnicityModalOpen
+      ? "hidden"
+      : "unset";
+  }, [isRaceEthnicityModalOpen]);
```
Comment (mxosman) [inline publisher/src/components/MetricsConfiguration/Configuration.tsx:61]: Nothing changed here - or in L45-46 aside from moving this from the child component to the parent component so the modal appears in the correct nesting structure with the new styling adjustments.

File: publisher/src/components/MetricsConfiguration/Configuration.tsx [hunk 3]
```diff
@@ -51,76 +54,79 @@ function Configuration() {
...
-            </Styled.MenuItemLabel>
-          </Styled.MenuItem>
-        </Styled.Menu>
-        {metricEnabled === null && (
-          <Styled.MetricIndicator isAlert>
-            <img src={indicatorAlertIcon} alt="" /> Configuration required
-          </Styled.MetricIndicator>
+    <Styled.MetricConfigurationContainer>
+      <Styled.MetricConfigurationWrapper>
+        <Styled.ButtonPositionWrapper>
```
Comment (mxosman) [inline publisher/src/components/MetricsConfiguration/Configuration.tsx:83]: Adjusted some of the wrapper stylings which caused different indents in this diff. No functionality changed here except for usage of the TabbedBar component and some minor naming updates.

**Review by morden35 — APPROVED** [review]

**Pull request merged**